src/translator.cc

```diff
@@ -129,6 +129,24 @@ namespace ctranslate2 {
     return results;
   }
 
+  void
```

Comment:
It looks like this function does not need to be a member of `Translator`. Can you make it a static function instead and remove the declaration in `translator.h`?

```cpp
static void replace_unknowns(...)
```

Reply:
Sure.